featuretools/tests/conftest.py

```diff
@@ -746,6 +747,52 @@ def rolling_outlier_series_pd():
     )
 
 
+@pytest.fixture
+def postal_code_dataframe():
```

Comment:
Our convention has been to use the suffix `_pd` to denote the pandas version of the fixture, and then we drop all suffixes in the fixture that gets used for the tests. So I would rename this to `postal_code_dataframe_pd` and rename the `postal_code_dataframes` fixture to `postal_code_dataframe`.